Patches/The GiantRace/Scenario/Giant_Scenario.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>The GiantRace</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ScenarioDef[defName="GI_Scenario"]/scenario/parts</xpath>
+				<value>
+					<li Class="ScenPart_StartingThing_Defined">
+						<def>StartingThing_Defined</def>
+						<thingDef>Ammo_Arrow_Stone</thingDef>
+						<count>100</count>

```

Comment:
Any idea why the scenario gives them bows in the first place? The giants can't use them.

Reply:
Scenario also gives you regular humans as well as giants, it's a mixed tribal start pretty much.